src/main/java/org/elasticsearch/index/query/FuzzyQueryBuilder.java

```diff
@@ -62,6 +62,14 @@ public FuzzyQueryBuilder(String name, Object value) {
     }
 
     /**
+     * package private constructor for serialization only
+     */
+    FuzzyQueryBuilder() {
+        this.name = null;
+        this.value = null;
```

Comment:
In general, I don't see any other way to do this at the moment, not loving though having to assign null to so many fields...although it's only a package private constructor that does it,  maybe we could move the instance to the builders, call them prototype and make the default constructor with null assignment even private rather than package private. Or even remove it and have the INSTANCE passing in null values during object creation. Thoughts?